eng/pipelines/templates/jobs/workload-build.yml

```diff
@@ -20,8 +20,10 @@ jobs:
     enableSbom: true
     artifacts:
       publish:
-        artifacts: true
-        logs: true
+        artifacts:
+          name: Artifacts
+        logs:
+          name: Logs
```

Comment:
Adjusted the artifact names here so tooling (like SBOM) knows that to expect. This removed the need for using the `Artifacts_Windows_NT_Release` name for artifacts, which doesn't make sense in this pipeline.